en/net/working-with-facades/pdfpageeditor/manipulate-page-properties/_index.md

```diff
@@ -101,4 +147,38 @@ The following code snippet shows how to resize the contents of some specific pag
 
 
 
-{{< gist "aspose-pdf" "4a12f0ebd453e7f0d552ed6658ed3253" "Examples-CSharp-AsposePdfFacades-Pages-ManipulatePageProperties-ResizePageContents-ResizePageContents.cs" >}}
+```csharp
+ private static void ResizePdfPageContents()
+ {
+     // The path to the documents directory
+     var dataDir = RunExamples.GetDataDir_AsposePdf();
+
+     // Create PdfFileEditor Object
+     var fileEditor = new Aspose.Pdf.Facades.PdfFileEditor();
```

Comment:
DO use using for disposable objects (streams, document, etc).

Reply:
PdfFileEditor is not disposable